extensions/blocks/story/story.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Story Block.
+ *
+ * @since 8.6.1
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Story;
+
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'story';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	jetpack_register_block(
+		BLOCK_NAME,
+		array( 'render_callback' => __NAMESPACE__ . '\render_block' )
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Render an image inside a slide
+ *
+ * @param array $media  Image information.
+ *
+ * @return string
+ */
+function render_image( $media ) {
+	if ( empty( $media['id'] ) || empty( $media['url'] ) ) {
+		return __( 'Error retrieving media', 'jetpack' );
+	}
+	return sprintf(
+		'<img
+			alt="%1$s"
+			class="wp-block-jetpack-story_image wp-story-image wp-image-%2$s"
+			data-id="%2$s"
+			src="%3$s">',
+		esc_attr( $media['alt'] ),
+		$media['id'],
```

Comment:
If the ID is an int, let's be explicit about it.

```suggestion
			class="wp-block-jetpack-story_image wp-story-image wp-image-%2$d"
			data-id="%2$d"
			src="%3$s">',
		esc_attr( $media['alt'] ),
		absint( $media['id'] ),
```